src/app/inventory/dimStoreItem.scss

```diff
@@ -97,6 +97,12 @@ dim-store-item {
     border: none;
     padding: 2px;
   }
+  &.masterwork {
+    border-color: $gold;
+  }
+  &.masterwork:hover {
+    box-shadow: inset 0 0 0 2px #DDD;
+  }
```

Comment:
Hmm, I'm not sure about the hover effect. Maybe we should just go with the gold border?

Reply:
Borders are... a bit tricksy. We have the gold (complete) border to ID modded items, but that doesn't read clearly on exotics. A purple border wouldn't read clearly on legendaries.

Masterwork in-game has a gold-white-gold gradient by default (I tried that and it was not noticeable at the size that we render) and, when you hover over it, it has a double-white border that goes outside of the item's box. I tried double borders with a CSS pseudo element, but couldn't get that to actually render in our display grid.

I'm fine with ditching the hover effect, but I know people will want something to ID masterwork items. The search filter's there in the near term.